pkg/limayaml/validate_test.go

```diff
@@ -75,13 +75,18 @@ func TestValidateMinimumLimaVersion(t *testing.T) {
 
 func TestValidateProbes(t *testing.T) {
 	images := `images: [{"location": "/"}]`
+	images2 := `images: [{"location": "/",digest: "69f31d3208895e5f646e345fbc95190e5e311ecd1359a4d6ee2c0b6483ceca03"}]`
 	validProbe := `probes: [{"script": "#!foo"}]`
 	y, err := Load([]byte(validProbe+"\n"+images), "lima.yaml")
 	assert.NilError(t, err)
-
 	err = Validate(y, false)
 	assert.NilError(t, err)
 
+	y2, err := Load([]byte(validProbe+"\n"+images2), "lima.yaml")
+	assert.NilError(t, err)
+	err = Validate(y2, false)
+	assert.Error(t, err, "field `images[0].digest` is invalid: 69f31d3208895e5f646e345fbc95190e5e311ecd1359a4d6ee2c0b6483ceca03: invalid checksum digest format")
+
```

Comment:
Better is to create a separate test: `TestValidateDigest(t *testing.T) {`.

It should cover these lines:

```
	if f.Digest != "" {
		if err := f.Digest.Validate(); err != nil {
			errs = errors.Join(errs, fmt.Errorf("field `%s.digest` is invalid: %s: %w", fieldName, f.Digest.String(), err))
			return errs
		}
		if !f.Digest.Algorithm().Available() {
			errs = errors.Join(errs, fmt.Errorf("field `%s.digest` refers to an unavailable digest algorithm", fieldName))
		}
	}
```